lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -281,6 +267,17 @@ static LogicalResult verifyCellOp(CellOp cell) {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// AssignOp
+//===----------------------------------------------------------------------===//
+static LogicalResult verifyAssignOp(AssignOp assign) {
+  auto parent = assign->getParentOp();
+  if (isa<GroupOp>(parent) || isa<WiresOp>(parent))
+    return success();
+  return assign.emitOpError(
+      "should only be contained in 'calyx.wires' or 'calyx.group'");
```

Comment:
Being nitpicky here, but if you can't capture this verifier logic in ODS, should probably add an error test to cover it.